cosim/black-parrot-example/fpga/Makefile

```diff
@@ -4,8 +4,11 @@ include ../Makefile.design
 # Accelerator Software Settings
 #############################
 NBF_FILE     ?= $(abspath ../prog.nbf)
-HOST_PROGRAM ?= $(abspath ../ps.cpp)
 SIM_ARGS     += $(NBF_FILE)
+HOST_PROGRAM ?= $(abspath ../ps.cpp)
+ifdef PK
+HOST_PROGRAM += $(abspath $(shell find ../htif/ -name '*.cc'))
```

Comment:
Rather than combining the .cc into the main compilation, it's better practice to compile a htif.o and link it to main

Reply:
htif uses zpl and ps address mapping, and ps instastiates an htif object, so they're pretty intertwined and I thought it's just easier to build them together.